### PR TITLE
Draft citation.

### PR DESCRIPTION
**Pull request by RubelMozumder**
- [x] Check why README.md has been changed.
- [ ]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-6. [Special Keywords in YAML](#special-keywords-in-yaml)
-   - [Keyword `exists`](#keyword-exists)
-   - [Keyword `unit`](#keyword-unit)
-   - [Keyword `dimensions`](#keyword-dimensions)
-   - [Keyword `enumeration`](#keyword-enumeration)
-   - [Keyword `xref`](#keyword-xref)
-7. [How to Install nyaml](#how-to-use-nyaml-tool)
-8. [Conclusion](#conclusion)
-9. [References](#references)
+- [Table of Contents](#table-of-contents)
```
Comment (lukaspie) [inline README.md:5]: Is this changed on purpose?

File: pyproject.toml [hunk 2]
```diff
@@ -43,7 +43,8 @@ dev = [
     "pre-commit",
     "types-pyyaml",
     "lxml-stubs",
-    "uv>=0.4.10"
+    "uv>=0.4.10",
+    "cffconvert", # validates CITATION.cff
```
Comment (lukaspie) [inline pyproject.toml:47]: can be removed, not in use

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  The `nyaml` tool also supports the reverse process, converting XML-based NXDL schemas into
+  YAML format. Users can benefit from a more accessible and human-readable way to define
+  NeXus data structures, while still adhering to NeXus standards. The tool requires users to
+  follow a specific grammar (see full documentation at https://fairmat-nfdi.github.io/nyaml/)
+  for embedding NeXus data structures, ensuring compatibility with the XML-based NXDL.
+
+  This work is part of the FAIRmat project (project ID 460197019), funded by the Deutsche
+  Forschungsgemeinschaft (DFG, German Research Foundation)
+  (https://gepris.dfg.de/gepris/projekt/460197019?language=en). The project aims to promote FAIR data principles—making scientific data Findable, Accessible, Interoperable, and Reusable—within the materials science community.
```
Comment (aalbino2) [inline CITATION.cff:57]:
```suggestion
  (https://gepris.dfg.de/gepris/projekt/460197019?language=en). Visit the [FAIRmat page](https://www.fairmat-nfdi.eu/fairmat/about-fairmat/consortium-fairmat) for more info. 
```

[the remark this answers is not in the document]
Comment (aalbino2) [reply CITATION.cff:57]: agree

**Review by aalbino2 — COMMENTED** [review]
Looks good, I chnaged the last sentence of the abstract.

File: README.md (hunk 1 above)
Comment (RubelMozumder) [reply README.md]: Right, it came from format on autosave.

Comment (RubelMozumder) [reply README.md]: Back to the previous version.

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,54 @@
+cff-version: 1.2.0
+title: A python package converts YAML to NXDL format and vice versa
```
Comment (lukaspie) [inline CITATION.cff:2]:
```suggestion
title: A Python package that converts YAML to NXDL format and vice versa
```

**Review by lukaspie — APPROVED** [review]
LGTM

**Pull request merged**